test/contlinear.jl

```diff
@@ -0,0 +1,98 @@
+# Continuous linear problems
+
+function contlineartest(solver::AbstractSolver, eps=Base.rtoldefault(Float64))
+    @testset "Set up model, variables, constraint, objective" begin
+        # simple 2 variable, 1 constraint problem
+        # min -x
+        # st   x + y <= 1   (x + y - 1 ∈ NonPositive)
+        #       x, y >= 0   (x, y ∈ NonNegative)
+
+        m = Model(solver)
+
+        v = addvariables!(m, 2)
+        @test getattribute(m, VariableCount()) == 2
+
+        @test cansetattribute(m, VariableLowerBound())
```

Comment:
We don't do ``VariableLowerBound`` anymore. We should add a variablewise constraint with the GreaterThan set.